content/2020/12-10-blank-oauth2-redirect-uri-change.md

```diff
@@ -0,0 +1,45 @@
+---
+applied_at: "2020-12-10"
+applies_to: 
+- api
+is_impactful: true
+is_new_feature: false
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Blank OAuth 2 redirect URI change
+
+On September 28th, 2020 we announced upcoming changes to our
+[security requirements for OAuth 2 app redirect URIs][oauth2-changelog-notice].
+As of today, applications that use a blank redirect URI will no longer be
+permitted, and will begin to produce an error when attempting to redirect the
+user, stating that there is a mismatched URI.
```

Comment:
quick comment - the message we're showing is redirect_uri missing if there is no saved redirect, not sure if that is important to note.

Reply:
Added the specific error code to the changelog. Thanks!